esp/esdllib/esdl_def.cpp

```diff
@@ -1464,6 +1466,7 @@ class EsdlDefinition : public CInterface, implements IEsdlDefinition
     unsigned walkChildrenDepthFirst( AddedObjs& foundByName, EsdlDefObjectWrapperArray& dependencies, IEsdlDefObject* esdlObj, double requestedVer, IProperties *opts, int level=0, unsigned flags=0);
     bool shouldGenerateArrayOf( IEsdlDefObject& child, unsigned flags, unsigned& stateOut );
     bool setWrapperFlagStringArray( AddedObjs& foundByName, EsdlDefObjectWrapper* wrapper, unsigned state );
+    virtual bool isShared() override { return IsShared(); }
```

Comment:
curious why need this and not just the underlying non-virtual existing IsShared() method?

Reply:
Because we want to avoid a dynamic_cast to CInterface*.